src/_help/continuous-integration/api.md

```diff
@@ -0,0 +1,59 @@
+---
+title: API
+---
+
+- TOC
+{:toc}
+
+If neither the [Github-Action](/help/continuous-integration/github-actions/) or the [CLI](/help/continuous-integration/cli/) are sufficient for you, please read this page to discover what you can do with the [Bump.sh API](https://developers.bump.sh).
```

Comment:
Is there any reason that could lead an user to go with the API? We could use that here, instead of the negative form?